package.json

```diff
@@ -114,6 +115,7 @@
   "devDependencies": {
     "@storybook/react": "4.0.0-alpha.8",
     "amphtml-validator": "^1.0.23",
+    "audit-ci": "^1.2.1",
```

Comment:
What was the reason for adding a dependency when we can run `npm audit` directly?